embed.go

```diff
@@ -0,0 +1,20 @@
+package teleport
```

Comment:
This is the missing copyright notice the linter is complaining about. You can run `make fix-license` to add it automatically.

Reply:
I did not know this. You learn something every day :-)